.github/workflows/deploy-pages.yml

```diff
@@ -3,16 +3,16 @@ name: Build and Deploy
 on:
   push:
     branches:
-      - master
+      - main
 
 jobs:
   build:
     if: github.repository == 'packit/packit.dev'
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout master
-        uses: actions/checkout@v1
+      - name: Checkout branch
+        uses: actions/checkout@v2
```

Comment:
@jpopelka I [see](https://github.com/actions/checkout#checkout-v2) 'v2' is the recommended version for this action. Was there any reason for using v1 specifically when you've initially set up this workflow?

Reply:
Because the [example @ the action's page](https://github.com/benmatselby/hugo-deploy-gh-pages#example) still says `v1`. Maybe even the action's author doesn't know about `v2`? :-)
Let's try your `v2` change and see.